README.md

```diff
@@ -25,8 +25,12 @@ We don't know how many people have COVID-19. So we made a website where people c
 - 🇨🇦 Canada: https://coronastatus.ca
 - 🇦🇺 Australia: https://coronastatusau.org
 - 🇸🇬 Singapore: https://coronastatus.sg
+- 🇦🇷 Argentina: https://coronastatus.com.ar
 - 🇲🇾 Malaysia: https://coronastatusmy.org
+<<<<<<< HEAD
```

Comment:
Little merge conflict here that needs to be resolved :)

Reply:
Yes I just say it. Do you know any trick to catch them before doing a push?

Reply:
When you try to merge master into your branch, you will be informed that you have merge conflicts. If you then run `git status`, you will see which files that have conflicts. These need to be resolved before running `git commit`.

Reply:
Thanks!